packages/Core/theme/modals.ts

```diff
@@ -25,15 +25,11 @@ export const getModals = (theme: WuiTheme): ThemeModals => {
       zIndex: 999,
     },
     footer: {
-      borderTopColor: colors.light[800],
-      borderTopStyle: 'solid',
-      borderTopWidth: borderWidths.sm,
+      borderTop: `${borderWidths.sm} solid ${colors.light[800]}`,
```

Comment:
borderTopStyle is not implemented in @xstyled/system 3.*
:eyes: https://github.com/gregberge/xstyled/pull/341